ibc_data/conditions.csv

```diff
@@ -0,0 +1,377 @@
+MVIS,2_dots_control,1 position to remember because of the highlighted dot
+MVIS,4_dots,4 positions to remember
+MVIS,4_dots_control,1 position to remember because of the highlighted dot
+MVIS,6_dots,6 positions to remember
+MVIS,6_dots_control,1 position to remember because of the highlighted dot
+Moto,finger_right,"Movement of the right index finger, indicated by a button-press"
+Moto,finger_left,"Movement of the left index finger, indicated by a button-press"
+Moto,foot_left,Movement of the left foot
+Moto,foot_right,Movement of the right foot
+Moto,hand_left,Movement of the left hand
+Moto,hand_right,Movement of the right hand
+Moto,saccade_left,Movement of the eyes to the left
+Moto,saccade_right,Movement of the eyes to the right
+Moto,tongue_left,Movement of the tongue to the left
+Moto,tongue_right,Movement of the tongue to the right
+Moto,fixation,Gaze fixation on the central square
+MCSE,high_salience_left,Effect of salience on the left side
+MCSE,high_salience_right,Effect of salience on the right side
+MCSE,low_salience_left,Effect of salience on the left side
+MCSE,low_salience_right,Effect of salience on the right side
+Audio,animal,Sound of animal noises
+Audio,music,Musical sound
+Audio,nature,Naturalistic sound
+Audio,speech,Human speech sound
+Audio,tool,Sound of tool usage
+Audio,voice,Non-speech human sound
+Audio,catch,Repetition of the previous sound
+Audio,silence,No sound
+Attention,double_congruent,"The stim is not cued, and all the distractors are congruent with the target"
+Attention,double_incongruent,"The stim not is cued, and all the distractors are incongruent with the target"
+Attention,double_cue,
+Attention,spatial_incongruent,"The stim is cued, and all the distractors are incongruent with the target"
+Attention,spatial_congruent,"The stim is cued, and all the distractors are congruent with the target"
+StopSignal,go,Answer to the stim
+StopSignal,stop,Hold motor response
+TwoByTwo,task_stay_cue_stay,Both the task and the cue are the same
+TwoByTwo,task_switch_cue_switch,Only the cue switches
+TwoByTwo,task_switch_cue_stay,"Task switch. Since the cues are divided in two groups, this means that the cue stays in the same group, but obviously changes since the task has changed"
+TwoByTwo,task_stay_cue_switch,"Task switch. Practically, this condition and the previous one are the same, but this time the cue also changes to the other group"
+Discount,delay,Effect of reward delay
+Discount,amount,Effect of reward gain
+SelectiveStopSignal,go_critical,Answer to the stim (critical side)
+SelectiveStopSignal,go_noncritical,Answer to the stim (noncritical side)
+SelectiveStopSignal,stop,Hold motor response
+SelectiveStopSignal,ignore,Answer regardless of the stop signal
+Stroop,congruent,Color and word are the same
+Stroop,incongruent,Color and word are different
+ColumbiaCards,num_loss_cards,Probability of losing
+ColumbiaCards,loss,Expected loss
+ColumbiaCards,gain,Expected gain
+DotPatterns,cue,

```

Comment:
Look at the stimulus to provide a description of the cue

Reply:
Integrated